CHANGELOG.md

```diff
@@ -62,6 +62,8 @@ These changes are available on the `master` branch, but have not yet been releas
   ([#2624](https://github.com/Pycord-Development/pycord/pull/2624))
 - Fixed `AttributeError` when accessing `Member.guild_permissions` for user installed
   apps. ([#2650](https://github.com/Pycord-Development/pycord/pull/2650))
+- Fixed `Interaction.channel` was malformed.
```

Comment:
that's not grammar... "Fixed malformed `Interaction.channel`" should satisfy the grammar requirement.

Maybe add a brief clause of information as to what the criteria would cause the malformation (e.g. "Fixed malformed `Interaction.channel` when accessed from hell").

Reply:
lmao

Reply:
Is it good like now ?

Reply:
I assume it is good since you didn't comment on it in the new review. Please unresolve if needed.